docs/outlook/delegate-access.md

```diff
@@ -79,7 +79,11 @@ The following example shows the `SupportsSharedFolders` element set to `true` in
 You can get an item's shared properties in Compose or Read mode by calling the [item.getSharedPropertiesAsync](../reference/objectmodel/preview-requirement-set/office.context.mailbox.item.md#methods) method. This returns a [SharedProperties](/javascript/api/outlook/office.sharedproperties) object that currently provides the delegate's permissions, the owner's email address, the REST API's base URL, and the target mailbox.
 
 > [!IMPORTANT]
-> In a delegate scenario, your add-in can use REST but not EWS, and the add-in's permission must be set to `ReadWriteMailbox` to enable REST access to the owner's mailbox.
+> - In a delegate scenario, your add-in can use REST but not EWS, and the add-in's permission must be set to `ReadWriteMailbox` to enable REST access to the owner's mailbox.
+> - In Message Compose mode, this API is not supported in Outlook on the web or Windows unless the following conditions are met.
+>    1. The owner shares their **Drafts** folder with the delegate.
+>    2. The delegate saves a draft message then moves it from their own **Drafts** folder to the shared **Drafts** folder.
+>    3. The delegate opens the draft from the shared **Drafts** folder.
```

Comment:
This makes it sound like only drafts folders are supported which is not true. I would suggest changing this to the following:
Any message compose(new email, reply, forward) is a new email item from your main mailbox and not the shared mailbox. Changing the from address on the compose form does not make it a shared item. A compose item is shared only when the folder in which this compose item resides is shared with you. 
Instead of the last line we could also use: Only when a compose item resides in a folder that is shared with you is the item considered as 'shared'.

Reply:
If you;d like we can also provide an example of this scenario